src/notification/email/mode/alert.pm

```diff
@@ -938,6 +939,11 @@ URL of the Centreon web interface. Use either HTTP or HTTPS protocol depending o
 --centreon-url='http://your-centreon-server'
 --centreon-url='https://your-centreon-server'
 
+=item B<--url-path>
+
+Pathname for Centreon (default: centreon) 
```

Comment:
default is /centreon (not centreon)

That makes me feel that maybe the code should check if the provided path starts with a / and either fail or add it if it's not the case.